File: 1012.py
a, b, c = input().split(" ")
triangulo = (float(a) * float(c))/2
circulo = 3.14159 * (float(c)* float(c))
trapezio = float(c) *(float(a) + float(b)) / 2
quadrado = float(b) * float(b)
retangulo = float(a) * float(b)
print(f'''TRIANGULO: {triangulo:.3f}
CIRCULO: {circulo:.3f}
TRAPEZIO: {trapezio:.3f}
QUADRADO: {quadrado:.3f}
RETANGULO: {retangulo:.3f}''')

File: 1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var linha1 = lines.shift().split(" ")
var a = Number(linha1[0]);var b = Number(linha1[1]);var c = Number(linha1[2]);
var triangulo = (a * c)/2; var circulo = 3.14159 * c * c; var trapezio = c * (a + b) / 2; var quadrado = b * b; var retangulo = a * b;
console.log(`TRIANGULO: ${triangulo.toFixed(3)}
CIRCULO: ${circulo.toFixed(3)}
TRAPEZIO: ${trapezio.toFixed(3)}
QUADRADO: ${quadrado.toFixed(3)}
RETANGULO: ${retangulo.toFixed(3)}`);
